### DISCUSSION TITLE
saml authentication with Microsoft Azure

### DISCUSSION THREAD

**Discussion by ashokthangaraj** [netbox-community/netbox#11944] [Help Wanted!]
Hi,
Can you please help us on sharing the values to be added in configuration.py and settings.py to enable microsoft azure saml authentication.

Thanks

**Comment by bogdancordos** [comment]
Hi,

The process is detailed here:
https://demo.netbox.dev/static/docs/administration/authentication/microsoft-azure-ad/

**Reply by ashokthangaraj** [reply]
@bogdancordos 
this is for OAuth. I am looking for SAML configuration.If you have configured please share the config's.

**Reply by F3d0s** [reply]
Have you tried with [django3_saml2_nbplugin](https://github.com/jeremyschulman/netbox-plugin-auth-saml2)?

**Reply by ashokthangaraj** [reply]
@F3d0s ,
Yes i tried but it is keep throwing dependency error with the latest version of netbox3.4.X

**Reply by CTV-2023** [reply]
Check here for a working exemple with social_core.backends.saml.SAMLAuth
[Reddit post](https://www.reddit.com/r/Netbox/comments/104ouqz/netbox_saml_with_azure_ad_authorization/)
